### PR TITLE
Implement HSM messages

### PR DESCRIPTION
**Pull request by guiajlopes**

### CODE REVIEW

File: src/MessageBird/Objects/Conversation/HSM.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     */
+    public $templateName;
+
+    /**
+     * @var HSMLanguage $language
+     */
+    public $language;
+
+    /**
+     * @var []HSMParams $params
```
Comment (modprobe) [inline src/MessageBird/Objects/Conversation/HSM.php]:
```suggestion
     * @var HSMParams[] $params
```

File: src/MessageBird/Objects/Conversation/HSMLanguage.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace MessageBird\Objects\Conversation;
+
+class HSMLanguage
```
Comment (modprobe) [inline src/MessageBird/Objects/Conversation/HSMLanguage.php]: Could we move the HSM* Classes into a sub-namespace? So e.g. `HSMCurrency -> MessageBird\Objects\Conversations\HSM\Currency`

File: src/MessageBird/Objects/Conversation/HSM.php (hunk 1 above)
Comment (guiajlopes) [reply src/MessageBird/Objects/Conversation/HSM.php]: fixed

File: src/MessageBird/Objects/Conversation/HSMLanguage.php (hunk 2 above)
Comment (guiajlopes) [reply src/MessageBird/Objects/Conversation/HSMLanguage.php]: thank you for your feedback, I've moved it to the HSM namespace :)

**Pull request merged**